dev-requirements.txt

```diff
@@ -1,5 +1,5 @@
-black
-flake8
-mypy
-pytest
-pytest-cov
+black==21.5b2
+flake8==3.8.4
+mypy==0.812
+pytest==6.2.3
+pytest-cov==2.12.1
```

Comment:
I was getting burned because I had a slightly older version of `black` locally, so it wasn't formatting the same way the latest version was and tests were failing. This is why I'm a big fan of pinning!